cmd/variables.go

```diff
@@ -70,6 +70,42 @@ func (h *Handler) VariablesSet(ctx context.Context, req *entity.CommandRequest)
 	fmt.Print(ui.Heading(fmt.Sprintf("Updated %s for \"%s\"", strings.Join(updatedEnvNames, ", "), environment.Name)))
 	fmt.Print(ui.KeyValues(*updatedEnvs))
 
+	err = h.maybeRedeployAfterVariablesChange(ctx, environment)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (h *Handler) maybeRedeployAfterVariablesChange(ctx context.Context, environment *entity.Environment) error {
```

Comment:
lol, can we rename this to `validateAndUpdateDeployment` or something like that. `maybe` just doesn't sit right here

Reply:
Haha, must be my time with Elm that made me appreciate `maybe`